packages/headless/src/features/commerce/context/context-actions-loader.ts

```diff
@@ -29,6 +31,14 @@ export interface ContextActionCreators {
    * @returns A dispatchable action.
    */
   setView(payload: SetViewPayload): PayloadAction<SetViewPayload>;
+
+  /**
+   * Sets the user context property without modifying any other context properties.
+   *
+   * @param payload - The action creator payload.
+   * @returns A dispatchable action.
+   */
+  setUser(payload: SetUserPayload): PayloadAction<SetUserPayload>;
```

Comment:
If you decide apply my comment about the name of the context controller method, I think the action should be renamed too.